mappings/net/minecraft/server/ServerConfigEntry.mapping

```diff
@@ -4,5 +4,5 @@ CLASS net/minecraft/class_3330 net/minecraft/server/ServerConfigEntry
 		ARG 1 key
 	METHOD method_14626 getKey ()Ljava/lang/Object;
 	METHOD method_14627 isInvalid ()Z
-	METHOD method_24896 fromJson (Lcom/google/gson/JsonObject;)V
+	METHOD method_24896 toJson (Lcom/google/gson/JsonObject;)V
```

Comment:
```suggestion
	METHOD method_24896 write (Lcom/google/gson/JsonObject;)V
```

https://github.com/FabricMC/yarn/blob/743dedee468d2f47fb31eab97f0cc1cedf385a91/CONVENTIONS.md#serialization

> Use "to"
for methods that convert an object to another type (`toString`, `toLong`, `toNbt`).

> Use "write" for methods that save data to an *existing* 
object passed as a parameter.